docs/docs/dev/policies/service-health-probes.md

```diff
@@ -0,0 +1,147 @@
+# Service Health Probes
+
+Kuma is able to track the status of the Envoy proxy. If grpc stream with Envoy is disconnected then Kuma considers this 
+proxy as offline, but we still send the traffic regardless of that, because this status is designed to track the connection
+between `kuma-cp` and `kuma-dp`. 
+
+Also, every `inbound` in the Dataplane model has `health` section:
+
+```yaml
+type: Dataplane
+mesh: default
+name: web-01
+networking:
+  address: 127.0.0.1
+  inbound:
+    - port: 11011
+      servicePort: 11012
+      health:
+        ready: true
+      tags:
+        kuma.io/service: backend
+        kuma.io/protocol: http
+```
+
+This `health.ready` status is intended to show the status of the service itself. It is set differently depending on 
+the environment ([Kubernetes](#kubernetes-probes) or [Universal](#universal-probes)), but it's treated the same way 
+regardless of the environment:
+
+- if `health.ready` is true or `health` section is missing - Kuma considers the inbound as healthy and includes it 
+  into load balancing set.
+- if `health.ready` is false -  Kuma doesn't include the inbound into load balancing set.
+
+Also, `health.ready` is used to compute the status of the Dataplanes and Service. You can see these statuses both in Kuma GUI and Kuma CLI:
+
+- if proxy status is `Offline`, then Dataplane is `Offline`:
+- if proxy status is `Online`:
+  - if all inbounds are ready then Dataplane is `Online`
+  - if all inbounds are not ready then Dataplane is `Offline`
+  - if at least one of the inbounds is not ready then Dataplane is `Partially degraded` 
+  - if inbound is not ready then it's not included in the load-balancer set which means it doesn't receive the traffic
+  - if all inbounds which implement the same service are ready then service is `Online`
+  - if all inbounds which implement the same service are not ready then service is `Offline`
+  - if at least one of the inbounds which implement the same service is not ready then service is `Partially degraded`
+
+## Kubernetes probes
+
+Even if Kubernetes probes are disabled, Kuma takes `pod.status.containerStatuses.ready` in order to fill `dataplane.inbound.health` section.
```

Comment:
This is useful info but might be better if it's moved further down (let's first describe what we do :) )